packages/ember-routing/lib/system/router.js

```diff
@@ -1563,4 +1564,9 @@ function representEmptyRoute(liveRoutes, defaultParentState, route) {
   }
 }
 
+deprecateProperty(EmberRouter.prototype, 'router', 'routerMicrolib', {
+  id: 'ember-router.router',
+  until: '2.16'
```

Comment:
Can you add the `url` property here (from the entry you are adding to the deprecations page on the website)?